yarn-project/noir-contracts/scripts/install_noirup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Script to install noirup and the latest nargo
+set -eu
+
+SPECIFIED_HOME=${1:-$HOME}
+VERSION="aztec"
```

Comment:
Why the `VERSION` here, seems unused?

Reply:
ah sorry auto merge rugged, it was stale, ill remove as part of a rebase